.github/workflows/scan_docker_image.yml

```diff
@@ -0,0 +1,63 @@
+name: Scan docker image
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run at 7:30AM UTC every day
```

Comment:
Is this standard? Do we need to discuss this setting as a team?

Reply:
Everywhere in LAA that does a container scan does it as a cron job (unless its during their deploy pipeline)
https://github.com/search?q=org%3Aministryofjustice%20github%2Fworkflows%20%2Fsnyk.yml&type=code
https://github.com/search?q=org%3Aministryofjustice+snyk%2Factions%2Fdocker&type=code

The idea being that you are checking the container for changes in vulnerabilities every day, even if you aren't pushing new code, because you want to be aware of any issues with services even if they are live and not getting heavily developed.

As for the time it runs, there is no real science behind that choice.